src/OrchardCore.Modules/OrchardCore.Workflows/Manifest.cs

```diff
@@ -19,7 +19,7 @@
 
 [assembly: Feature(
     Id = "OrchardCore.Workflows.Http",
-    Name = "HTTP Workflows Activities",
+    Name = "HTTP Workflow Activities",
```

Comment:
"Workflows" as in the module's name?

Reply:
For HTTP it was "Workflows" but for the Timer it is "Workflow"

    Name = "Timer Workflow Activities",

So i opted to use "Workflow" everywhere "Workflow activities" is used

But as you want

Reply:
Okay, i will put a `s` on both ;)